README.md

```diff
@@ -58,9 +75,9 @@ class Application extends React.Component {
 - `edgeHitWidth` (Number) - Edge distance on content view to open side menu, defaults to 60
 - `toleranceX` (Number) - X axis tolerance
 - `toleranceY` (Number) - Y axis tolerance
-- `disableGestures` (Bool) - Disable whether the menu can be opened with gestures or not
-- `onStartShouldSetResponderCapture` (Function) - Function that accepts event as an argument and specify if side-menu should react on the touch or not. Check https://facebook.github.io/react-native/docs/gesture-responder-system.html for more details
-- `onChange` (Function) - Callback on menu open/close. Is passed `isOpen` as an argument
+- `onSwipe` (Function) - Function that handles gestures, receives boolean argument indicating whether menu should be opened or not based on the drag. When not defined, gestures are disabled
```

Comment:
> Do you think it's a transparent behavior if function is not defined - assume that we don't want to enable gestures?

That's what we do now. If that function is not defined, gestures are disabled. 

> Why do you think we need to give user a change if menu should be opened or not? I think it can be configured by the offsets user can setup.

Because that PR removes `this.isOpen` from component completely. Side menu has no internal state nor does not store any informations about whether it's open or not. It has to use a callback so that parent component updates state and `this.props.isOpen` reflects new position.